src/subarrayn.jl

```diff
@@ -0,0 +1,25 @@
+const unsafe_getindex = Base.unsafe_getindex
```

Comment:
That was just in the original code, so I left it in, didn't know if it was necessary. I'll remove it.